### PR TITLE
[Feature] Statistics analyze task support kill statement

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/qe/ConnectScheduler.java [hunk 1]
```diff
@@ -204,4 +204,51 @@ public void run() {
             }
         }
     }
+
+    public boolean submit2(ConnectContext context) {
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/qe/ConnectScheduler.java:208]: Give a more meaning name.

File: fe/fe-core/src/main/java/com/starrocks/qe/ConnectScheduler.java [hunk 2]
```diff
@@ -204,4 +204,51 @@ public void run() {
...
+            return false;
+        }
+        context.setConnectionId(nextConnectionId.getAndAdd(1));
+        context.resetConnectionStartTime();
+        context.setThreadLocalInfo();
+        context.setConnectScheduler(ConnectScheduler.this);
+        return true;
+    }
+
+    private class LoopHandler2 implements Runnable {
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/qe/ConnectScheduler.java:219]: Give a more meaning name.

**Comment by kangkaisen** [comment]
Need to add some test cases.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 3 / 31 (09.68%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|0|1|00.00%|[1825]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|0|1|00.00%|[515]|
|:large_blue_circle:|com/starrocks/statistic/SampleStatisticsCollectJob.java|0|1|00.00%|[59]|
|:large_blue_circle:|com/starrocks/statistic/StatisticExecutor.java|0|4|00.00%|[201, 202, 203, 211]|
|:large_blue_circle:|com/starrocks/statistic/FullStatisticsCollectJob.java|0|1|00.00%|[52]|
|:large_blue_circle:|com/starrocks/statistic/StatisticsCollectJob.java|0|2|00.00%|[85, 88]|
|:large_blue_circle:|com/starrocks/sql/ast/KillAnalyzeStmt.java|0|7|00.00%|[11, 12, 13, 16, 21, 26, 31]|
|:large_blue_circle:|com/starrocks/statistic/HistogramStatisticsCollectJob.java|0|1|00.00%|[67]|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeManager.java|1|7|14.29%|[267, 268, 271, 272, 273, 275]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|5|20.00%|[474, 866, 867, 868]|
|:large_blue_circle:|com/starrocks/sql/ast/ShowAnalyzeStatusStmt.java|1|1|100.00%|[]|

**Comment by HangyuanLiu** [comment]
> Need to add some test cases.

done

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 16 / 33 (48.48%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/statistic/SampleStatisticsCollectJob.java|0|1|00.00%|[59]|
|:large_blue_circle:|com/starrocks/statistic/StatisticExecutor.java|0|5|00.00%|[202, 203, 204, 205, 213]|
|:large_blue_circle:|com/starrocks/statistic/FullStatisticsCollectJob.java|0|1|00.00%|[52]|
|:large_blue_circle:|com/starrocks/statistic/StatisticsCollectJob.java|0|2|00.00%|[87, 88]|
|:large_blue_circle:|com/starrocks/statistic/HistogramStatisticsCollectJob.java|0|1|00.00%|[67]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|5|20.00%|[474, 866, 867, 868]|
|:large_blue_circle:|com/starrocks/sql/ast/KillAnalyzeStmt.java|4|7|57.14%|[16, 21, 31]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeManager.java|9|9|100.00%|[]|

**Review by kangkaisen — DISMISSED** [review]

**Review by Seaven — DISMISSED** [review]

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 16 / 33 (48.48%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/statistic/SampleStatisticsCollectJob.java|0|1|00.00%|[59]|
|:large_blue_circle:|com/starrocks/statistic/StatisticExecutor.java|0|5|00.00%|[202, 203, 204, 205, 213]|
|:large_blue_circle:|com/starrocks/statistic/FullStatisticsCollectJob.java|0|1|00.00%|[52]|
|:large_blue_circle:|com/starrocks/statistic/StatisticsCollectJob.java|0|2|00.00%|[87, 88]|
|:large_blue_circle:|com/starrocks/statistic/HistogramStatisticsCollectJob.java|0|1|00.00%|[67]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|5|20.00%|[464, 826, 827, 828]|
|:large_blue_circle:|com/starrocks/sql/ast/KillAnalyzeStmt.java|4|7|57.14%|[16, 21, 31]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeManager.java|9|9|100.00%|[]|

**Review by Seaven — DISMISSED** [review]

**Review by kangkaisen — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 16 / 33 (48.48%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/statistic/SampleStatisticsCollectJob.java|0|1|00.00%|[59]|
|:large_blue_circle:|com/starrocks/statistic/StatisticExecutor.java|0|5|00.00%|[202, 203, 204, 205, 213]|
|:large_blue_circle:|com/starrocks/statistic/FullStatisticsCollectJob.java|0|1|00.00%|[52]|
|:large_blue_circle:|com/starrocks/statistic/StatisticsCollectJob.java|0|2|00.00%|[87, 88]|
|:large_blue_circle:|com/starrocks/statistic/HistogramStatisticsCollectJob.java|0|1|00.00%|[67]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|1|5|20.00%|[464, 834, 835, 836]|
|:large_blue_circle:|com/starrocks/sql/ast/KillAnalyzeStmt.java|4|7|57.14%|[16, 21, 31]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeManager.java|9|9|100.00%|[]|

**Review by kangkaisen — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Pull request merged**